gallery/visualtests/nxBreadcrumb.js

```diff
@@ -36,6 +36,7 @@ describe('NxBreadcrumb', function() {
     );
 
     await dropdownBtn.click();
+    await moveMouseAway();
```

Comment:
Added because the button was showing its hover state instead of selected state

Reply:
> This has created a problem in the snapshots - if we use moveMouseAway(), then the tooltip begins to disappear when the snapshot is taken, or if we remove moveMouseAway() the color of the icon only button is in the hover state color. Is there a preference as to which we should use?

I'd remove it.  The color of the icon button isn't really the point of this screenshot; it's more about the custom width of the dropdown.